distribution/docs/src/main/resources/content/_configuring/importing-exported-configurations-contents.adoc

```diff
@@ -6,20 +6,21 @@
 
 ==== Importing the Exported Configuration Settings
 
-[WARNING]
-====
-Importing of configurations must take place instead of performing installation for the first time and after performing OS-level hardening.
-====
-
 To import a previously exported configuration file from the ${command-console}:
 
+. Step through the installation process.
 . Launch the newly installed ${branding}.
```

Comment:
❗️  Whoops. Can't step through the installer if the system isn't even started yet. Did we mean "Restart" the newly installed ${branding}?